R/restrict_derivation.R

```diff
@@ -0,0 +1,92 @@
+#' Execute a Derivation on a Subset of the Input Dataset
+#'
+#' Execute a derivation on a subset of the input dataset.
```

Comment:
Remove line 3, its a repeat of Line 1.

Reply:
If I remove line 3, the line is repeated in the rendered documentation but both lines are in title case. This looks a bit odd. Thus I would keep it.